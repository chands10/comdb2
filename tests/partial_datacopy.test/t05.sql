create table t {
    schema {
        int a
        int b
        int c
        int d
    }
    keys {
        datacopy(a) "a" = a
    }
};$$

create table t2 {
    schema {
        int a
        int b
        int c
        int d
    }
    keys {
        datacopy(a, b, c, d) "abcd" = a + b + c + d
    }
};$$

select * from comdb2_keys where tablename='t';
select * from comdb2_partial_datacopies where tablename='t';

select * from comdb2_keys where tablename='t2';
select * from comdb2_partial_datacopies where tablename='t2';

drop table t;
drop table t2;
