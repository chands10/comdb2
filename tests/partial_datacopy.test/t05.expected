(tablename='t', keyname='A', keynumber=0, isunique='Y', isdatacopy='N', ispartialdatacopy='Y', isrecnum='N', condition=NULL, uniqnulls='N')
(tablename='t', keyname='A', columnname='a')
(tablename='t2', keyname='ABCD', keynumber=0, isunique='Y', isdatacopy='N', ispartialdatacopy='Y', isrecnum='N', condition=NULL, uniqnulls='N')
(tablename='t2', keyname='ABCD', columnname='a')
(tablename='t2', keyname='ABCD', columnname='b')
(tablename='t2', keyname='ABCD', columnname='c')
(tablename='t2', keyname='ABCD', columnname='d')
